### PR TITLE
Translation for the page 'Handling Events'

### LINKED ISSUE #20: Translation for the page 'Basics -> Handling Events'
# Please help us translating this page in Italian!
  
  **Aiutaci a tradurre questa pagina in Italiano!**

  You can find all the details about how to help here: https://github.com/reactjs/it.reactjs.org/issues/1
  
  ## GRAZIE! :smile:

### PR DESCRIPTION
**Pull request by lssmn**
Fixes #20

### CODE REVIEW

File: content/docs/handling-events.md [hunk 1]
```diff
@@ -91,64 +91,64 @@ ReactDOM.render(
...
-        Click me
+        Clicca qui
       </button>
     );
   }
 }
 ```
 
-The problem with this syntax is that a different callback is created each time the `LoggingButton` renders. In most cases, this is fine. However, if this callback is passed as a prop to lower components, those components might do an extra re-rendering. We generally recommend binding in the constructor or using the class fields syntax, to avoid this sort of performance problem.
+Il problema con questa sintassi è che viene creato un callback diverso ogni volta che `LoggingButton` viene renderizzato. Nella maggior parte dei casi, non vi sono problemi. Tuttavia, se questo callback viene passato come prop a componenti inferiori, tali componenti potrebbero eseguire un ulteriore re-renderizzamento. In generale, vi consigliamo di utilizzare `bind` nel costruttore o la sintassi delle proprietà pubbliche nelle classi, per evitare questo tipo di problema di prestazioni.
```
Comment (deblasis) [inline content/docs/handling-events.md]:
```suggestion
Il problema con questa sintassi è che viene creata una callback diversa ogni volta che `LoggingButton` viene renderizzato. Nella maggior parte dei casi, non vi sono problemi. Tuttavia, se questa callback viene passata come prop a componenti inferiori, tali componenti potrebbero eseguire un ulteriore re-renderizzamento. In generale, vi consigliamo di utilizzare `bind` nel costruttore o la sintassi delle proprietà pubbliche nelle classi, per evitare questo tipo di problema di prestazioni.
```
The subject is the "callback function", which in Italian should be feminine I guess

File: content/docs/handling-events.md [hunk 2]
```diff
@@ -91,64 +91,64 @@ ReactDOM.render(
...
 
-## Passing Arguments to Event Handlers {#passing-arguments-to-event-handlers}
+## Passare Argomenti ai Gestori di Eventi {#passing-arguments-to-event-handlers}
 
-Inside a loop it is common to want to pass an extra parameter to an event handler. For example, if `id` is the row ID, either of the following would work:
+All'interno di un ciclo, è comune avere l'esigenza di passare un parametro aggiuntivo a un gestore di eventi. Ad esempio, avendo `id` come l'identificativo della riga, le seguenti dichiarazioni sarebbero entrambe valide:
```
Comment (deblasis) [inline content/docs/handling-events.md]:
```suggestion
All'interno di un ciclo, è comune avere l'esigenza di passare un parametro aggiuntivo ad un gestore di eventi. Ad esempio, avendo `id` come l'identificativo della riga, le seguenti dichiarazioni sarebbero entrambe valide:
```

File: content/docs/handling-events.md (hunk 1 above)
Comment (lssmn) [reply content/docs/handling-events.md]: That's right 👍

**Comment by deblasis** [comment]
Good job @lssmn ! 👍

**Pull request merged**